doc/developer-notes.md

```diff
@@ -1089,11 +1089,11 @@ A few guidelines for introducing and reviewing new RPC interfaces:
     new RPC is replacing a deprecated RPC, to avoid both RPCs confusingly
     showing up in the command list.
 
-- Use *invalid* bech32 addresses (e.g. the constant `EXAMPLE_ADDRESS`) for
+- Use *invalid* addresses (e.g. in the constant array `EXAMPLE_ADDRESS`) for
   `RPCExamples` help documentation.
 
   - *Rationale*: Prevent accidental transactions by users and encourage the use
-    of bech32 addresses by default.
+    of example addresses by default.
```

Comment:
That doesn't make sense. "Encourage the use of..." adresses the _user_ of the software here, not the developer. As above, "bech32" should stay here.